src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherAccountingLineTotalsValidation.java

```diff
@@ -57,12 +57,15 @@ public boolean validate(AttributedDocumentEvent event) {
                     return true;
                 }
                 // KFSMI- 5183
-                if (persistedDocument.getDocumentHeader().getWorkflowDocument().isSaved() && persistedDocument.getDisbVchrCheckTotalAmount().isZero()) {
+                if (persistedDocument.getDocumentHeader().getWorkflowDocument().isSaved() && persistedDocument
+                        .getDisbVchrCheckTotalAmount().isZero()) {
                     return true;
                 }
 
                 // check total cannot decrease
-                if (!persistedDocument.getDocumentHeader().getWorkflowDocument().isCompletionRequested() && (!persistedDocument.getDisbVchrCheckTotalAmount().equals(dvDocument.getDisbVchrCheckTotalAmount()))) {
+                if (!persistedDocument.getDocumentHeader().getWorkflowDocument()
+                        .isCompletionRequested() && persistedDocument.getDisbVchrCheckTotalAmount().equals(
```

Comment:
It looks like the old code was doing a not-equals check instead of an equals check, so this line will need further adjustment or maybe the isLessThan() was correct. Sorry for noticing this earlier.

Reply:
Thank you for catching that! I have updated the code.